src/algebra/module/prod.lean

```diff
@@ -14,28 +15,6 @@ variables {R : Type*} {S : Type*} {M : Type*} {N : Type*}
 
 namespace prod
 
-instance [has_scalar R M] [has_scalar R N] : has_scalar R (M × N) := ⟨λa p, (a • p.1, a • p.2)⟩
-
-@[simp] theorem smul_fst [has_scalar R M] [has_scalar R N]
-  (a : R) (x : M × N) : (a • x).1 = a • x.1 := rfl
-@[simp] theorem smul_snd [has_scalar R M] [has_scalar R N]
-  (a : R) (x : M × N) : (a • x).2 = a • x.2 := rfl
-@[simp] theorem smul_mk [has_scalar R M] [has_scalar R N]
-  (a : R) (b : M) (c : N) : a • (b, c) = (a • b, a • c) := rfl
-
-instance [has_scalar R S] [has_scalar S M] [has_scalar R M] [has_scalar S N] [has_scalar R N]
-  [is_scalar_tower R S M] [is_scalar_tower R S N] : is_scalar_tower R S (M × N) :=
-⟨λ x y z, mk.inj_iff.mpr ⟨smul_assoc _ _ _, smul_assoc _ _ _⟩⟩
-
-instance [has_scalar R M] [has_scalar S M] [has_scalar R N] [has_scalar S N]
-  [smul_comm_class R S M] [smul_comm_class R S N] :
-  smul_comm_class R S (M × N) :=
-{ smul_comm := λ r s x, mk.inj_iff.mpr ⟨smul_comm _ _ _, smul_comm _ _ _⟩ }
-
-instance {r : monoid R} [mul_action R M] [mul_action R N] : mul_action R (M × N) :=
-{ mul_smul  := λ a₁ a₂ p, mk.inj_iff.mpr ⟨mul_smul _ _ _, mul_smul _ _ _⟩,
-  one_smul  := λ ⟨b, c⟩, mk.inj_iff.mpr ⟨one_smul _ _, one_smul _ _⟩ }
-
 instance {r : monoid R} [add_monoid M] [add_monoid N]
   [distrib_mul_action R M] [distrib_mul_action R N] : distrib_mul_action R (M × N) :=
```

Comment:
Why did you move the other instances but not this one?

Reply:
Done.